skimage/segmentation/_chan_vese.py

```diff
@@ -0,0 +1,298 @@
+    tol : float, positive, optional
+        'energy' variation tolerance between iterations. If the
+        difference of 'energy' between two iterations is below this
+        value, the algorithm will assume that the solution was
+        reached.
+    maxiter : uint, optional
+        Maximum number of iterations allowed before the algorithm
+        interrupts itself.
+    dt : float, optional
+        A multiplication factor applied at calculations for each step,
+        serves to accelerate the algorithm. While higher values may
+        speed up the algorithm, they may also lead to convergence
+        problems.
+    starting_level_set : str or (M, N) 2D array, optional
+        Defines the starting level set used by the algorithm.
+        If a string is inputted, a level set that matches the image
+        size will automatically be generated. Alternatively, it is
+        possible to define a custom level set, which should be an
+        array of float values, with the same size as 'img'.
+        Accepted String values are as follows.
+
+        'checkerboard'
+            the starting level set is defined as
+            sin(x/5*pi)*sin(y/5*pi), where x and y are pixel
+            coordinates. This level set has fast convergence, but may
+            fail to detect implicit edges.
+        'disk'
+            the starting level set is defined as the opposite
+            of the distance from the center of the image minus half of
+            the minimum value between image width and image height.
+            This is somewhat slower, but is more likely to properly
+            detect implicit edges.
+        'small disk'
+            the starting level set is defined as the
+            opposite of the distance from the center of the image
+            minus a quarter of the minimum value between image width
+            and image height.
+    extended_output : bool, optional
+        If set to True, the return value will be a tuple containing
+        the three return values (see below). If set to False which
+        is the default value, only the 'segmentation' array will be
+        returned.
+
+    Returns
+    -------
+    phi : (M, N) 2D array, float
+        Final level set computed by the algorithm.
+    segmentation : (M, N) 2D array, bool
+        Segmentation produced by the algorithm.
+    energies : list of floats
+        Shows the evolution of the 'energy' for each step of the
+        algorithm. This should allow to check whether the algorithm
+        converged.
+
+    References
+    ----------
+    .. [1] An Active Contour Model without Edges, Tony Chan and
+           Luminita Vese, Scale-Space Theories in Computer Vision,
+           1999
+    .. [2] Chan-Vese Segmentation, Pascal Getreuer Image Processing On
+           Line, 2 (2012), pp. 214-224.
+    .. [3] The Chan-Vese Algorithm - Project Report, Rami Cohen
+           http://arxiv.org/abs/1107.2782 , 2011
+    """
```

Comment:
Maybe add a Notes section to the docstring to say which energy is minimized? This might help the user select the values of the parameters.